ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -282,3 +282,24 @@ export async function validateDocumentID (documentId: string) {
     }
   }
 }
+
+export async function submitMhrTransfer (payloadData, mhrNumber) {
+  const url = sessionStorage.getItem('MHR_API_URL')
+  const config = { baseURL: url, headers: { Accept: 'application/json' } }
```

Comment:
For the POST `/mhr/api/v1/transfers/{mhrNumber}` api call, **Account-Id** is required in the headers.

Reply:
Oh! I didn't see, do you have a reference?

Reply:
Sent you via RocketChat.

Reply:
@dimak1 Sorry, to follow up on this, the `account-id` IS included. 
It's handled by the axios interceptor `axios-ppr.ts`. It is conditionally included in header config when there is a `currentAccount` present. ie any non-staff, for the most part. 

Good eye though, thanks for noting this!

Reply:
@doug-lovett Oh, you beat me to it 😆 
Thank you!